tests/test-cfg.json

```diff
@@ -0,0 +1,4 @@
+{
+    "authToken": "FDLRTQAMNGR3EX2PUYRP2VM6CP35GQC2",
```

Comment:
Yikes.

Reply:
Yeah, I'm a dummy. That file's not even supposed to be checked in.

Token's been removed, invalidated, expunged, and other forceful words.